src/element/bounds.ts

```diff
@@ -482,9 +507,17 @@ const getLinearElementRotatedBounds = (
 export const getElementBounds = (
   element: ExcalidrawElement,
   recompuate = false,
-): [number, number, number, number] => {
-  if (element.bounds && !recompuate) {
-    return element.bounds;
+): Bounds => {
+  if (!recompuate) {
+    const cachedBounds = cachedElementBounds.getCachedBounds(element);
+    if (
+      cachedBounds &&
+      cachedBounds.version === element.version &&
+      cachedBounds.versionNounce === element.versionNonce &&
```

Comment:
using either `version` or `versionNonce` should be fine (`version` is an incrementing integer, `versionNonce` is a random number — used for collab reconciliation purposes but should work fine). Also it's nonce not _nounce_ :p